elastic_monte_carlo_estimate_of_pi.ipynb

```diff
@@ -0,0 +1,358 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Monte-Carlo Estimate of $\\pi$\n",
+    "\n",
+    "We want to estimate the number $\\pi$ using a [Monte-Carlo method](https://en.wikipedia.org/wiki/Pi#Monte_Carlo_methods) exploiting that the area of a quarter circle of unit radius is $\\pi/4$ and that hence the probability of any randomly chosen point in a unit square to lie in a unit circle centerd at a corner of the unit square is $\\pi/4$ as well.  So for N randomly chosen pairs $(x, y)$ with $x\\in[0, 1)$ and $y\\in[0, 1)$, we count the number $N_{circ}$ of pairs that also satisfy $(x^2 + y^2) < 1$ and estimage $\\pi \\approx 4 \\cdot N_{circ} / N$.\n",
+    "\n",
+    "[<img src=\"https://upload.wikimedia.org/wikipedia/commons/8/84/Pi_30K.gif\" \n",
+    "     width=\"50%\" \n",
+    "     align=top\n",
+    "     alt=\"PI monte-carlo estimate\">](https://en.wikipedia.org/wiki/Pi#Monte_Carlo_methods)\n",
+    "\n",
+    "## Why Adaptive?\n",
+    "\n",
+    "Using [Dask's adaptivity](http://docs.dask.org/en/latest/setup/adaptive.html), we'll show that it is possible to scale the available resources to meet almost any desired wall times irrespective of the actual work load.  This is important, because it allows for focussing on the situation of the human running the analysis:\n",
+    "\n",
+    "> _\"Am I in an explorative and creative phase of my work where it is important that I can see the next plot within seconds?\"_\n",
+    "\n",
+    "> _\"Am I running routine analyses that can wait until tomorrow or next week?\"_\n",
+    "\n",
+    "## Actual timings\n",
+    "\n",
+    "Aming for a duration of 20 seconds per calculation, this is what we actually get:\n",
```

Comment:
Typo in Aiming.
Maybe put the results at the end? To give the user an idea of what can be achieved, even if he probably won't scale all the way up.